src/components/Accordion/__test__/accordion.spec.js

```diff
@@ -0,0 +1,399 @@
+        secondAccordionSectionButton.simulate('click');
+
+        expect(handleToggleSectionMockFn.mock.calls[0][1]).toEqual(['accordion-test-2']);
+    });
+
+/* when any <AccordionSection /> is expanded by default */
+    it('should set aria-expanded to "true" in "button" when click in AccordionSection', () => {
+        const component = mount(
+            <Accordion>
+                <AccordionSection label="Rainbow Accordion">
+                    AccordionSection-1
+                </AccordionSection>
+            </Accordion>,
+        );
+        const button = component.find('button.rainbow-accordion-section_summary-button-heading');
+        button.simulate('click');
+        const buttonClicked = component.find('button.rainbow-accordion-section_summary-button-heading');
+
+        expect(buttonClicked.prop('aria-expanded')).toBe(true);
+    });
+    it('should set isExpanded to "true" in "ArrowIcon" when click in AccordionSection', () => {
+        const component = mount(
+            <Accordion>
+                <AccordionSection label="Rainbow Accordion">
+                    AccordionSection-1
+                </AccordionSection>
+            </Accordion>,
+        );
+        const button = component.find('button.rainbow-accordion-section_summary-button-heading');
+        button.simulate('click');
+        const ArrowIcon = component.find('RightArrow');
+
+        expect(ArrowIcon.prop('isExpanded')).toBe(true);
+    });
+    it('should set aria-hidden to "false" in "container of the content" when click in AccordionSection', () => {
+        const component = mount(
+            <Accordion>
+                <AccordionSection label="Rainbow Accordion">
+                    AccordionSection-1
+                </AccordionSection>
+            </Accordion>,
+        );
+        const button = component.find('button.rainbow-accordion-section_summary-button-heading');
+        button.simulate('click');
+        const contentContainer = component.find('div.rainbow-accordion-section_content');
+
+        expect(contentContainer.prop('aria-hidden')).toBe(false);
+    });
+    it('should set isTrue to "true" in "container of the content`s RenderIf" when click in AccordionSection', () => {
+        const component = mount(
+            <Accordion>
+                <AccordionSection label="Rainbow Accordion">
+                    AccordionSection-1
+                </AccordionSection>
+            </Accordion>,
+        );
+        const button = component.find('button.rainbow-accordion-section_summary-button-heading');
+        button.simulate('click');
+        const accordionSection = component.find('section').childAt(1);
+        const renderIf = accordionSection.find('RenderIf');
+
+        expect(renderIf.prop('isTrue')).toBe(true);
+    });
+
+/* when <AccordionSection /> is expanded by default */
+    it('should set aria-expanded to "false" in "button" when click in an expanded AccordionSection', () => {
```

Comment:
same ^^ no click for this and put it below the first that test the aria-expanded to be true